sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -260,16 +260,16 @@ object DataSourceV2Strategy extends Strategy with PredicateHelper {
         Nil
       }
 
-    case DescribeTable(r: DataSourceV2Relation, isExtended) =>
-      DescribeTableExec(r.table, isExtended) :: Nil
+    case r @ DescribeTable(datasource: DataSourceV2Relation, isExtended) =>
+      DescribeTableExec(r.output, datasource.table, isExtended) :: Nil
```

Comment:
Nit: I'd prefer leaving the relation as `r` and using a better name for the `DescribeTable` node, like `desc`.

Reply:
OK. Thanks for pointing it out. Changed it to `desc`.